images/ceph/Makefile

```
@@ -121,3 +123,15 @@ csv: $(OPERATOR_SDK) $(YQ) ## Generate a CSV file for OLM.
 
 csv-clean: $(OPERATOR_SDK) $(YQ) ## Remove existing OLM files.
 	@rm -fr ../../cluster/olm/ceph/deploy/* ../../cluster/olm/ceph/templates/*
+
+# list-image creates list of images for offline installation
+list-image:
+	@echo "producING list of images for offline installation";\
+	# remove the file if already exists
+	[ -f ../../cluster/examples/kubernetes/ceph/rook-ceph-image.txt ] && rm ../../cluster/examples/kubernetes/ceph/rook-ceph-image.txt;\
```

Let's go with a simple filename since the path has enough context
```suggestion
	[ -f ../../cluster/examples/kubernetes/ceph/images.txt ] && rm ../../cluster/examples/kubernetes/ceph/images.txt;\
```